scripts/build-sass.sh

```diff
@@ -1,15 +1,20 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I need to learn why `#!/bin/bash` works but `#!/bin/sh` errors. See: https://stackoverflow.com/a/23277604/2194699